holoviews/util.py

```diff
@@ -28,14 +28,17 @@ class Dynamic(param.ParameterizedFunction):
     kwargs = param.Dict(default={}, doc="""
         Keyword arguments passed to the function.""")
 
+    shared_data = param.Boolean(default=False, doc="""
+        Whether the cloned DynamicMap will share the same data.""")
```

Comment:
``.data`` on a DynamicMap is not the callback but the cache. Will update the docstring to reflect that.